databook/physiology/stimuli/passive-visual-stimuli/visual-stimuli-list.md

```diff
@@ -2,16 +2,16 @@
 
```

Comment:
I have an open ticket to rewrite this section so it's not worth editing it yet. This is fine, but just for future you can leave it be until that's done.